generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.tsx.ejs

```diff
@@ -67,6 +67,7 @@ export default class Header extends React.Component<IHeaderProps, IHeaderState>
 
   <%_ if (enableTranslation) { _%>
   handleLocaleChange = event => {
+    Storage.session.set('locale', event.target.value);
```

Comment:
lets assign `event.target.value` to a const and use that everywhere in the method